op-batcher/Dockerfile

```diff
@@ -1,32 +1,9 @@
-FROM --platform=$BUILDPLATFORM golang:1.21.1-alpine3.18 as builder
-
-ARG VERSION=v0.0.0
-
-RUN apk add --no-cache make gcc musl-dev linux-headers git jq bash
-
-COPY ./go.mod /app/go.mod
-COPY ./go.sum /app/go.sum
-
-WORKDIR /app
-
-RUN go mod download
-
-# build op-batcher with the shared go.mod & go.sum files
-COPY ./op-batcher /app/op-batcher
-COPY ./op-bindings /app/op-bindings
-COPY ./op-node /app/op-node
-COPY ./op-service /app/op-service
-
-COPY ./.git /app/.git
-
-WORKDIR /app/op-batcher
-
-ARG TARGETOS TARGETARCH
-
-RUN make op-batcher VERSION="$VERSION" GOOS=$TARGETOS GOARCH=$TARGETARCH
+ARG OP_STACK_GO_BUILDER=us-docker.pkg.dev/oplabs-tools-artifacts/images/op_stack_go:latest
```

Comment:
This pattern means that `docker build -f op-batcher/Dockerfile .` is completely broken at the moment & not likely what the developer wants (because it's pulling the latest tag, not building the docker image).

Doing a shared build & multi-step build seems really nice, but this behavior is pretty weird.